src/factories/deckCreator.test.js

```diff
@@ -0,0 +1,12 @@
+import createDeck from "./deckCreator";
+const deck = createDeck();
+
+describe("createDeck", () => {
+  it("should create a deck", () => {
```

Comment:
Não é necessário colocar `should` no início de cada teste, todo código "should do something", então isso fica implicito.
```
Should raise an exception => Raises an exception
Should return true => Returns true
Should call #shuffle => Calls #shuffle
```

```suggestion
  it("creates a deck", () => {
```

> Do not use should when describing your tests. Use the third person in the present tense.
Ref: https://www.betterspecs.org/#should

Reply:
Ok! :fist_oncoming: